workspaces/vanilla/.env.sample

```diff
@@ -1,8 +1,8 @@
-tre_id="1234"
-workspace_id="001"
-address_space="10.2.2.0/24"
+tre_id="marcus-dev1"
+workspace_id="006"
+address_space="10.2.6.0/24"
 location=westeurope
 AZURE_TENANT_ID=_CHANGEME_
 AZURE_SUBSCRIPTION_ID=_CHANGEME_
-AZURE_CLIENT_ID=_CHANGEME_
-AZURE_CLIENT_SECRET=_CHANGEME_
\ No newline at end of file
+AZURE_SERVICE_PRINCIPAL_CLIENT_ID=_CHANGEME_
+AZURE_SERVICE_PRINCIPAL_CLIENT_PASSWORD=_CHANGEME_
```

Comment:
```suggestion
AZURE_CLIENT_ID=_CHANGEME_
AZURE_CLIENT_SECRET=_CHANGEME_
```

Reply:
As much as like the long version (self-documenting!), the official convention seem to have become this. See e.g., https://docs.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential

Reply:
:) I changed that, but reverted back as is what you had used. Will change back. :)